### PR TITLE
BXMSDOC-3775 Installing and configuring DM 7.3 and PAM 7.3 on JWS

### PR DESCRIPTION
**Pull request by emmurphy1**
[RHPAM draft](http://file.ork.redhat.com/~emmurphy/rhpam-7.3-jws)
[RHDM draft](http://file.ork.redhat.com/~emmurphy/rhdm-7.3-jws)

Added valve definition section to zip installation instructions.

### CODE REVIEW

**Comment by sutaakar** [comment]
@emmurphy1 Please add there informations about downloading and extracting offline Maven repo and RHDM Core Engine distribution

**Review by domhanak — APPROVED** [review]
Looks good, let's wait for @sutaakar to review maven repo part.

File: doc-content/enterprise-only/installation/jws-zip-install-proc.adoc [hunk 1]
```diff
@@ -10,38 +10,81 @@ To use the installer JAR file to install {KIE_SERVER}, see <<assembly_installing
...
 .. Unzip the `{PRODUCT_FILE}-add-ons.zip` file. The
 `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip`
  file is in the unzipped directory.
-. Extract the 
-ifdef::PAM[]
-`{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip`
+
+. Download the Decision Manager offline Maven repository and Decision Engine:
+.. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+ * *Product:* Decision Manager
```
Comment (sutaakar) [inline doc-content/enterprise-only/installation/jws-zip-install-proc.adoc]: Several occurrences of "Decision Manager" in PAM document.
Also duplicated downloading of add-ons zip.

File: doc-content/enterprise-only/installation/jws-zip-install-proc.adoc [hunk 2]
```diff
@@ -10,38 +10,81 @@ To use the installer JAR file to install {KIE_SERVER}, see <<assembly_installing
...
+.. Unzip the `rdhm-{PRODUCT_VERSION_LONG}-add-ons.zip` file.
+.. From the unzipped `rdhm-{PRODUCT_VERSION_LONG}-add-ons.zip` file, extract the  `rhdm-{PRODUCT_VERSION}-decision-engine.zip` file.
+ 
 endif::PAM[]
-ifdef::DM[]
-`{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip`
-endif::DM[]
- archive to a temporary directory. In the following examples, this directory is called `__TEMP_DIR__`.
++
+In the following examples, the directory that contains the extracted `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip` file is called `JWS_TEMP_DIR` and the directory that contains the extracted `rhdm-{PRODUCT_VERSION}-decision-engine.zip` file is called `DECISION_ENGINE`.
```
Comment (sutaakar) [inline doc-content/enterprise-only/installation/jws-zip-install-proc.adoc]: DECISION_ENGINE needs to be replaces with process engine variant

File: doc-content/enterprise-only/installation/jws-zip-install-proc.adoc [hunk 3]
```diff
@@ -10,38 +10,81 @@ To use the installer JAR file to install {KIE_SERVER}, see <<assembly_installing
...
 endif::DM[]
  directory to the `_JWS_HOME_/tomcat/webapps` directory.
 +
-WARNING: Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the {JWS} instance.
+[NOTE]
+====
+Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the {JWS} instance.
+====
 . Remove the `.war` extensions from the `kie-server.war` folder.
+. Move the `kie-tomcat-integration-{PRODUCT_VERSION}.jar` file from the `DECISION_ENGINE` directory to the `_JWS_HOME_/tomcat/lib` directory.
```
Comment (sutaakar) [inline doc-content/enterprise-only/installation/jws-zip-install-proc.adoc]: Same as above

File: doc-content/enterprise-only/installation/jws-zip-install-proc.adoc [hunk 4]
```diff
@@ -10,38 +10,81 @@ To use the installer JAR file to install {KIE_SERVER}, see <<assembly_installing
...
+. Move the `jboss-jacc-api_1.5_spec-{PRODUCT_VERSION}.jar`, `slf4j-api-{PRODUCT_VERSION}.jar`, and `slf4j-jdk14-{PRODUCT_VERSION}.jar` files from the `DECISION_ENGINE/lib` directory to the `_JWS_HOME_/tomcat/lib` directory.
```
Comment (sutaakar) [inline doc-content/enterprise-only/installation/jws-zip-install-proc.adoc]: Same as above

**Comment by sutaakar** [comment]
@emmurphy1 
DM looks good
PAM issues in comments

File: doc-content/enterprise-only/installation/controller-jws-install-proc.adoc [hunk 5]
```diff
@@ -6,15 +6,11 @@ You can configure {KIE_SERVER} to run in managed or unmanaged mode. If {KIE_SERV
 The {CONTROLLER} is integrated with {CENTRAL}. If you install {CENTRAL}, use the *Execution Server* page to create and maintain KIE containers. However, if you do not install {CENTRAL}, you can install the {HEADLESS_CONTROLLER} and use the REST API or the {KIE_SERVER} Java Client API to interact with it.
 
 .Prerequisites
+* The *{PRODUCT} {PRODUCT_VERSION_LONG} Add Ons* (`{PRODUCT_FILE}-add-ons.zip`) file has been been downloaded, as described in <<install-download-proc_{context}>>.
```
Comment (sterobin) [inline doc-content/enterprise-only/installation/controller-jws-install-proc.adoc]: "has been been" typo

File: doc-content/enterprise-only/installation/jws-zip-install-proc.adoc [hunk 6]
```diff
@@ -1,47 +1,69 @@
 [id='jws-zip-install-proc']
 
-= Installing {KIE_SERVER} with a ZIP file
+= Installing {KIE_SERVER} from ZIP files
 
-{KIE_SERVER} provides the runtime environment for business assets and accesses the data stored in the assets repository (knowledge store). You can use the `{PRODUCT_INIT}-{PRODUCT_VERSION}-kie-server-jws.zip` file to install {KIE_SERVER} on an existing {JWS} {JWS_VERSION_LONG} or higher server instance.
+{KIE_SERVER} provides the runtime environment for business assets and accesses the data stored in the assets repository (knowledge store). You can use zip files to install {KIE_SERVER} on an existing {JWS} {JWS_VERSION_LONG} or higher server instance.
```
Comment (sterobin) [inline doc-content/enterprise-only/installation/jws-zip-install-proc.adoc]: "ZIP" - Should be capped per our style. Search all. You go back and forth a bit I think. There's actually more to this story that I spoke with Ingrid about back when I first started here. It should be ZIP, but I think there's some kind of proprietary thing with that name such that using it this way is a violation of some kind. Not a matter for this PR as much as a global one that I was just reminded of. Will see what we do.

File: doc-content/enterprise-only/installation/run-dc-standalone-proc.adoc [hunk 7]
```diff
@@ -7,13 +7,11 @@ You can use the {CENTRAL} standalone JAR file to run {CENTRAL} without needing t
 Red Hat supports this installation type only when it is installed on Red Hat Enterprise Linux.
 ====
 
-.Procedure
+.Prerequisites
+* The *{PRODUCT} {PRODUCT_VERSION_LONG} {CENTRAL} Standalone* (`{PRODUCT_FILE}-{URL_COMPONENT_CENTRAL}-standalone.jar`) file has been been downloaded, as described in <<install-download-proc_{context}>>.
```
Comment (sterobin) [inline doc-content/enterprise-only/installation/run-dc-standalone-proc.adoc]: "has been been" typo again.

**Review by sterobin — COMMENTED** [review]
@emmurphy1 Done. Just a few small things.

**Pull request merged**